roles/display_deployment_plan/tasks/main.yml

```diff
@@ -2,64 +2,27 @@
 - name: Display deployment plan if interactive prompts are allowed
   when: not ((skip_interactive_prompts | default(false)) | bool)
   block:
+
     - name: Display inventory details and ask for user confirmation
       pause:
-        prompt: |-
-          An OpenShift cluster is about to be deployed. Please double check the provided inventory details.
-
-          * General configuration
-
-            {{ row_format_str | format('Cluster name', inventory.cluster_name) }}
-            {{ row_format_str | format('Base DNS domain', inventory.base_dns_domain) }}
-
-            {{ row_format_str | format('API Virtual IP', inventory.api_vip) }}
-            {{ row_format_str | format('Ingress Virtual IP', inventory.ingress_vip) }}
-
-            {{ row_format_str | format('OpenShift version', inventory.openshift_full_version) }}
-
-          * Prerequisite services configuration
-
-            {{ row_format_str | format('Setup NTP Service', inventory.setup_ntp_service) }}
-            {{ row_format_str | format('Setup DNS Service', inventory.setup_dns_service) }}
-            {{ row_format_str | format('Setup Registry Service', inventory.setup_registry_service) }}
-            {{ row_format_str | format('Setup HTTP Store Service', inventory.setup_http_store_service) }}
-
-          * Groups and hosts configuration
-
-          {% for group_name in inventory.groups_filtered %}
-            {{ group_name }}:
-            {% for host_name in groups[group_name] %}
-            {{ hosts_row_format_str | format(host_name, (hostvars[host_name]['ansible_host'] | default(messages.value_missing))) }}
-            {% else %}
-            {{ messages.hosts_missing }}
-            {% endfor %}
-
-          {% else %}
-            {{ messages.groups_missing }}
-
-          {% endfor -%}
-          ---
-
-          Are you sure you want to proceed with the deployment using this inventory configuration?
-
-          Enter a value [yes/NO]
+        prompt: "{{ lookup('template', 'plan.j2').strip('\n') }}"
       vars:
         # Every row consists of two columns. To ensure all items in the right column are aligned,
         # a minimum width of the left column is set in the following variable.
         left_column_width: 26
-        row_format_str: '%-{{ left_column_width }}s %s'
+        row_format_str: "%-{{ left_column_width }}s %s"
```

Comment:
Do we have a convention on strings style for Crucible? Out of habit, I've been using single quotes `'` to describe machine-strings, and double quotes `"` for strings intended for humans. 

Not sure if we've converged on a specific set of rules for that. 
Some discussion about the _machine-string/human-string_ convention can be found [here](https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python).

Reply:
Oh sorry I think an auto-formatter did that. I thought I had turned it off. I think that machine/human makes sense although I don't think we use that consistently. I like the idea though. I'll try to track down what causes the change to see if its a ansible convention.

Reply:
I changed it back and its not happening again. It must of happened before my auto formatter was turned off. I'll put it back to single quotes